src/main/scala/higherkindness/rules_scala/workers/zinc/common/ColorMessage.scala

```diff
@@ -0,0 +1,28 @@
+package higherkindness.rules_scala
+package workers.zinc.common
+
+import xsbti.Problem
+import Console.{RED => CR, YELLOW => CY, GREEN => CG, RESET => CRESET}
+
+class ColorMessage() {
```

Comment:
Let's just make this a singleton `object`.

Reply:
Good idea, fixed.